rotkehlchen/tests/api/blockchain/test_base.py

```diff
@@ -1462,13 +1405,8 @@ def _remove_blockchain_accounts_test_start(
             eth_balances=eth_balances_after_removal,
             token_balances=token_balances_after_removal,
             also_btc=True,
-            expected_liabilities=after_liabilities,
         )
 
-    # check that after removing ethereum account defi balances were updated
-    if query_balances_before_first_modification:
```

Comment:
hmmm don't you still need to check that the balances are no longer there? They are not defi balances anymore but should be normal balances too right? After removing the account they should be deleted right?

Reply:
unless I'm mistaken is the if above this one. For some reason the condition was being checked twice

Reply:
not sure. I guess maybe this was checking the in-memory defi balances which no longer exist so you can't keep it anyway.